packages/frontend-api/src/Resources/config/graphql-types/ChangePasswordInputDecorator.types.yaml

```diff
@@ -0,0 +1,32 @@
+ChangePasswordInputDecorator:
+    type: input-object
+    decorator: true
+    config:
+        fields:
+            email:
+                type: "String!"
+                description: "Customer user email."
+                validation:
+                    -   NotBlank:
+                            normalizer: "trim"
+                            message: "Please enter your email address"
+            oldPassword:
+                type: "String!"
+                description: "Current customer user password."
+                validation:
+                    -   NotBlank:
+                            normalizer: "trim"
+                            message: "Please enter old password"
+                    -   Length:
+                            min: 6
```

Comment:
Is it really necessary to check minimal length for old passwords? Cannot there exist any galaxy far far away where they allows shorter password?

Reply:
If there is some1 that allows shorter password, he should also update all constrains of it as it is checked in a about 7 places right now.

Reply:
When something is wrong does not mean you have to do it wrong all the time just to be consistent 😞
Leaving as is, but not really ok with that.